css/style.css

```diff
@@ -0,0 +1,1154 @@
+  height: 21.6rem;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  list-style: none;
+}
+
+.skill-list-items ul li {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 2.2rem;
+  width: max-content;
+  margin: 0.9rem 0;
+  margin-left: 2rem;
+  align-self: center;
+  line-height: 4.3rem;
+  color: #fff;
+}
+
+.skill-list-items ul li::before {
+  content: "";
+}
+
+.contact-section {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: absolute;
+  top: 330rem;
+  width: 105.7rem;
+  height: 55rem;
+  margin: auto 15rem;
+}
+
+.contact-heading {
+  width: 32.7rem;
+  height: 21.6rem;
+  margin: auto;
+}
+
+.contact-heading h3 {
+  position: relative;
+  top: -4.7rem;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 3.2rem;
+  line-height: 4.4rem;
+  margin: 0.3rem auto;
+  text-align: center;
+  color: #172b4d;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  width: 32.7rem;
+  height: 33rem;
+  margin: auto;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+}
+
+.form input {
+  width: 100%;
+  height: 4.8rem;
+  padding: 0.3rem;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1.5rem;
+  color: #979493;
+  border: none;
+  border-bottom: 0.1rem solid #979493;
+  padding-left: 0.8rem;
+  margin: 0.8rem 0;
+}
+
+#textarea {
+  width: 100%;
+  height: 11.4rem;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1.5rem;
+  line-height: 2.4rem;
+  color: #3c3a39;
+  border: none;
+  padding: 0.8rem;
+  background: #fbf8f7;
+  margin: 0.8rem 0;
+}
+
+.form-button {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.7rem;
+  color: #fff;
+  padding: 1.2rem;
+  border: 0.1rem solid var(--primary-color);
+  background: var(--primary-color);
+  width: 13.1rem;
+  height: 4.8rem;
+  position: relative;
+  top: 1.8rem;
+  margin: auto;
+}
+
+.form-button:hover {
+  background: #ffbe90;
+  color: #3b2b2b;
+}
+
+.footer-container {
+  position: absolute;
+  top: 385rem;
+  width: 100%;
+  height: 7.2rem;
+}
+
+.footer-icon-list {
+  width: 30rem;
+  margin: 1.8rem auto;
+}
+
+.footer-icon-list ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  list-style: none;
+}
+
+.footer-icon {
+  width: 3.4rem;
+  height: 1.6rem;
+  font-size: 2.5rem;
+  color: #505f79;
+  margin-right: 1.3rem;
+}
+
+.footer-icon:hover {
+  color: #347dfa;
+}
+
+.footer-vectorimg {
+  width: 3.4rem;
+  height: 2.6rem;
+  cursor: pointer;
+  font-size: 2rem;
+}
+
+.footer-vectorimg:hover {
+  filter:
+    invert(35%) sepia(21%) saturate(653%) hue-rotate(179deg)
+    brightness(93%) contrast(84%);
+}
+
+.left-navbar a:hover {
+  color: blueviolet;
+}
+
+.right-navbar ul li a:hover {
+  color: blueviolet;
+}
+
+@media screen and (max-width: 1324px) {
```

Comment:
- [ ] One of the objectives for this project is to use the mobile first approach, which means you start by styling the mobile view then you create media queries for desktop view. Kindly change all your media queries from `max-width` to `min-width` to satisfy that requirement 👍 
- [ ] The desktop version for your website starts at about `575px` which is too small (the acceptable values range between `768px` and `1440px`, Kindly fix this issue as well 💯 

![image](https://user-images.githubusercontent.com/50721479/213795096-fb08bcbb-6ad0-4a85-a929-712b9ed4e439.png)